srcpkgs/vault/files/vault/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec chpst -u _vault vault server -config=/etc/vault.hcl
```

Comment:
And run as `_vault` user here.
Am I missing something?

Reply:
shouldn't it have a user and group _vault ?

chpst -u _vault:_vault

Reply:
Leaving the group unset should be fine.  Running as nogroup would be acceptable if the umask is set such that files are only written 0600.

Reply:
`chpst -u _vault` is fine. It will default to group `_vault`

Reply:
sgtm.